src/components/UI/AddPost/AddPost.jsx

```diff
@@ -99,7 +99,7 @@ const AddPost = () => {
                         <span className='hover'><IconEmojiInput/></span>
                         <span className='hover'><IconGifInput/></span>
                     </div>
-                    <span onClick={AddPostHandler}><NormalButton name="Post" color="red"/></span>
+                    <span onClick={AddPostHandler}><NormalButton name="Post" class="post-btn hover-btn" color="red"/></span>
```

Comment:
click handler should be on buttons instead of span tag

Reply:
it is a button the Normal Button component is the button wrapped in a span